test/backward_compatibility/check_backward_compatibility.py

```diff
@@ -54,6 +54,8 @@
     ("aten::set_", datetime.date(2021, 1, 31)),
     ("aten::native_layer_norm", datetime.date(2021, 1, 31)),
     ("aten::native_layer_norm_backward", datetime.date(2021, 1, 31)),
+    ("aten::sort", datetime.date(2021, 1, 31)),
```

Comment:
Are these updates necessary even though the current signatures of sort aren't changing?

Reply:
Ah, yes, we can remove these now. No longer needed.